amundsen_application/static/js/components/DashboardPage/index.spec.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import * as History from 'history';
+
+import { shallow } from 'enzyme';
```

Comment:
is this file mainly used for testing purpose?

Reply:
Yes, files with this extension are unit test files for the React components. In our repo, we've chosen to have the test file in the same directory as the component code.